packages/server/lambdas/alfresco/searchQueryBuilder/luceneQueryBuilder.ts

```diff
@@ -19,6 +19,8 @@ const mimeTypesMapping = {
   ],
   TXT: ['"text/plain"'],
 };
+// Exported for testing, not meant for general usage
+export const mimeTypesMappingForTests = mimeTypesMapping;
```

Comment:
Why don't you import `mimeTypesMapping` directly in test file❓

Reply:
As the mimeTypesMapping is meant solely for "internal" use in luceneQueryBuilder, I wanted to emphasise that is exported only to be used for running unit tests.